src/js/window_pen.js

```
@@ -386,112 +399,58 @@ export class PenSystem extends ToolWindow {
-                var index = this.setSize(size);
-                if (!index) {
-                    // %1の太さは変更できません。
-                    this.axpObj.msg('@CAU0203', name);
-                    return;
+        quickSizeBtns.forEach((btn, i) => {
+            const saved = this.axpObj.configSystem.getConfig('QSIZE_' + i);
+            if (saved !== null && saved !== undefined) btn.dataset.size = saved;
+            renderQuickBtn(btn);
+
+            let lastTapTime = 0;
+            btn.addEventListener('pointerup', (e) => {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)